src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const plus = '+';
+  const minus = '-';
+  const times = 'X';
+  const divide = '/';
+  const modulo = '%';
+  const mNumberOne = Big(numberOne);
```

Comment:
**[OPTIONAL]**
You could add ```-/+``` to enable toggling a number from negative to positive vice versa.